### ISSUE TITLE
Train Visual Quality Discriminator model with real data and hyperparameter tuning

### ISSUE THREAD

**Issue by BasicallyOk** [BasicallyOk/deepfake-lip-sync#4]
Definition of Done: 

- Can test the model against test data and get an accuracy
- Can give a prediction for a given image that is not in the train or test set

**Comment by BasicallyOk** [comment]
Model is practically guessing. 
Made some changes, need to validate

**Comment by BasicallyOk** [comment]
Going to be done as soon as #17 is done

**Comment by BasicallyOk** [comment]
This ticket is fairly irrelevant without context of the GAN, moving to milestone GAN

**Comment by BasicallyOk** [comment]
Closing ticket. Too general.
